metadata.rb

```diff
@@ -4,11 +4,12 @@
 license 'Apache-2.0'
 description 'Installs the Chef Push Jobs Client'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '5.1.1'
+version '5.1.7'
```

Comment:
@iennae understood, this was an artifact of my local testing. I'll be closing this PR in favor of:   https://github.com/chef-cookbooks/push-jobs/pull/129 as it is more complete and I have been working with @zts on this anyways.